onetl/connection/db_connection/mongodb/connection.py

```diff
@@ -504,6 +504,16 @@ def write_df_to_target(
             else "append"
         )
 
+        if self._collection_exists(target):
+            if write_options.if_exists == MongoDBCollectionExistBehavior.ERROR:
+                raise ValueError("Operation stopped due to if_exists set to 'error'.")
+            elif write_options.if_exists == MongoDBCollectionExistBehavior.IGNORE:
+                log.info(
+                    "|%s| No further action is taken due to if_exists is set to 'ignore'",
```

Comment:
```suggestion
                    "|%s| No further action is taken due to 'if_exists' is set to 'ignore'",
```
тут тоже подсказать бы где искать if_exists

Reply:
обновил, сейчас поправлю то же самое в PR c Hive.WriteOptions